### PR TITLE
Added tutorial links for instruction 00, 01, 02

### PR DESCRIPTION
**Pull request by LijieZhou**

### CODE REVIEW

File: 00/README.md [hunk 1]
```diff
@@ -9,4 +9,6 @@ lead is called *anode* and needs to be connected to the positive voltage while t
 (called *cathode*) needs to be connected towards ground. If the LED is connected in reverse the LED
 will not light up.
 
+Tutorial: https://www.youtube.com/watch?v=GcfJXjz86n0&index=4&t=4s&list=PL2prrfB4uRarl12GHSbsoMmeYYlk2PLb7
+
```
Comment (apuder) [inline 00/README.md]: I suggest to embed the link like this:
Link to <a href="https://youtube.com/...">video tutorial</a>.

I also suggest to add those links at the bottom of each respective page. Please change for the three videos you have made so far.

**Review by apuder — CHANGES_REQUESTED** [review]
Please reformat the link as I suggested in my comment. Also, rebase to origin/master and get rid of commit b90f824. Change the comments for each commit to indicate for which tutorial this commit is for (e.g., "Add link for tutorial 00"). Then force-push for a clean history. Thx.

**Comment by LijieZhou** [comment]
I have sent another pull request with new changes based on your comments.

**Comment by apuder** [comment]
Thanks for your contribution. Looking forward to the next videos. :)

File: 00/README.md [hunk 2]
```diff
@@ -10,3 +10,5 @@ lead is called *anode* and needs to be connected to the positive voltage while t
 will not light up.
 
 <img src="doc/LEDResistor_bb.png" width="50%"/>
+
+Tutorial [link](https://www.youtube.com/watch?v=GcfJXjz86n0&list=PL2prrfB4uRarl12GHSbsoMmeYYlk2PLb7&t=4s&index=4)
```
Comment (shaeberling) [inline 00/README.md:14]: &t=4s will start the video 4 seconds int, cutting of part of the narrative. That's probably not what you want?

File: 01/README.md [hunk 3]
```diff
@@ -9,3 +9,4 @@ the same rules for the resistor and the polarity of the LED apply as discussed e
 
 <img src="doc/LEDResistorButton_bb.png" width="50%"/>
 
+Tutorial [link](https://www.youtube.com/watch?v=QSwEKkHwWP4&list=PL2prrfB4uRarl12GHSbsoMmeYYlk2PLb7&t=0s&index=3)
```
Comment (shaeberling) [inline 01/README.md:12]: "&t=0s" does nothing (sets an offset of 0)

File: 02/README.md [hunk 4]
```diff
@@ -35,3 +35,5 @@ is pressed or not and
 
 <img src="doc/ArduinoLEDButton_bb.png" width="50%"/>
 
+Tutorial [link](https://www.youtube.com/watch?v=qr91fY0JsOU&list=PL2prrfB4uRarl12GHSbsoMmeYYlk2PLb7&t=0s&index=2)
```
Comment (shaeberling) [inline 02/README.md:38]: Same here, "&t=0s" does nothing.

**Comment by apuder** [comment]
@LijieZhou: I think @shaeberling made some valid comments, especially regarding the t=4s. On second thought I would also suggest to embed the YouTube videos instead of adding a link. Why don't you create a new PR with those fixes for the first three videos?

**Comment by LijieZhou** [comment]
@apuder Did some research on embeded video on YouTube, and seems like it is not currently supported (except the github.io page). However, I can embed an clickable image to take the reader to the video. 
https://stackoverflow.com/questions/11804820/embed-a-youtube-video
Does that work? If so, I will use this trick and fix the glitch pointed out by @shaeberling

**Comment by apuder** [comment]
@LijieZhou the procedure posted on SO is a good compromise. Please make those changes. You can put them all in one PR with three commits for each of the three tutorials. Thx.

**Pull request merged**